electron/src/hardware-wallet.js

```diff
@@ -117,7 +117,30 @@ ipcMain.on('hwSendPin', (event, pin) => {
   if (pin) {
     lastPinPromiseResolve(pin);
   } else {
-    lastPinPromiseReject(new Error("Cancelled"))
+    lastPinPromiseReject(new Error("Cancelled"));
+  }
+});
+
+let lastPassphrasePromiseResolve;
+let lastPassphrasePromiseReject;
+
+const passphraseEvent = function() {
+  return new Promise((resolve, reject) => {
+    lastPassphrasePromiseResolve = resolve;
+    lastPassphrasePromiseReject = reject;
+
+    console.log("Hardware wallet passphrase requested");
+    if (win) {
+      win.webContents.send('hwPassphraseRequested');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163608)
```suggestion
      win.webContents.send("hwPassphraseRequested");
```